lib/mail_room/structured_logger.rb

```diff
@@ -0,0 +1,19 @@
+require 'logger'
+require 'json'
+
+module MailRoom
```

Comment:
It would be nice if there was a `NoopLogger` or something for a no-cost option for anyone who didn't enable logging.

Reply:
Done! I've made it so that the logger just swallows everything silently if there was no `structured_logger_file_name` attribute provided in the yml for that mailbox.